### DISCUSSION TITLE
queries regarding image conversion and quality

### DISCUSSION THREAD

**Discussion by hash-akansha** [ImageMagick/ImageMagick#2343] [Help]
1. Can RMagic will convert png to jpeg?
2. what are Size and Quality comparison between the compressed version of png/jpeg vs webp?

**Comment by snibgo** [comment]
ImageMagick can read PNG, and write JPEG.

I don't know RMagick. I assume it is a complete interface to ImageMagick.

PNG gives good compression for graphic images (eg solid colours). PNG compression is lossless. PNG can be 8 or 16 bits/channel/pixel. JPEG gives good compression for photos, but compression is lossy. JPEG is 8 bits/channel/pixel.

WEBP can be lossy or lossless. I don't have comparisons for compression or quality with PNG or JPEG.
